src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx

```diff
@@ -39,7 +39,6 @@ import type { SequenceType } from 'src/content/app/tools/blast/types/blastSettin
 
 import styles from './BlastInputSequences.scss';
 import sequenceBoxStyles from './BlastInputSequence.scss';
-
```

Comment:
could you undo this please? `git checkout dev src/content/app/tools/blast/components/blast-input-sequences/BlastInputSequencesHeader.tsx`?

Reply:
That's done